WhiteDentalClinic/WhiteDentalClinic.Application/Services/AppointmentService.cs

```diff
@@ -0,0 +1,103 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using WhiteDentalClinic.Application.Models.AppointmentModel;
+using WhiteDentalClinic.Application.Models.Dentist;
+using WhiteDentalClinic.Application.Services.Interfaces;
+using WhiteDentalClinic.DataAccess.Entities.AppointmentEntity;
+using WhiteDentalClinic.DataAccess.Entities.DentistEntity;
+using WhiteDentalClinic.DataAccess.Repositories.AppointmentRepository;
+
+namespace WhiteDentalClinic.Application.Services
+{
+    public class AppointmentService : IAppointmentService
+    {
+        private readonly IAppointmentRepository _appointmentRepository;
+        private readonly IMapper _mapper;
+        public AppointmentService(IAppointmentRepository appointmentRepository, IMapper mapper)
+        {
+            _appointmentRepository= appointmentRepository;
+            _mapper= mapper;
+        }
+        public IEnumerable<AppointmentResponseModel> GetAllAppointments()
+        {
+            var allAppointments = _appointmentRepository.GetAll();
+            return _mapper.Map<IEnumerable<AppointmentResponseModel>>(allAppointments);
+        }
+        public IEnumerable<AppointmentResponseModel> GetAllAppointmentsByCustomer(Guid customerRequestId)
```

Comment:
Add newlines between the methods, please. It's quite complicated to scan (read it quick and shallow just to see what this class does) the code this way.